File: GO/DDD_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestDDD(t *testing.T) {
	tables := []struct {
		cod  int
		want string
	}{
		{11, "Sao Paulo"},
		{13, "DDD nao cadastrado"},
	}

	for _, table := range tables {
		got := dddCity(table.cod)

		if got != table.want {
			t.Errorf("Waiting for this %s but the return was this: %s", table.want, got)
		}
	}
}

File: JS/DDD.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./DDD.js')

TEST('Starting DDD test...', (t) => {
    //Test 1
    t.assert(OBJ.dddCity(11) === "Sao Paulo", "Expect: 11 = Sao Paulo")
    //Test 2
    t.assert(OBJ.dddCity(13) === "DDD nao cadastrado", "Expect: 13 = DDD nao cadastrado")

    t.end()
})
